resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/minikube/Minikube.scala

```diff
@@ -76,7 +76,7 @@ private[spark] object Minikube extends Logging {
 
   def getDockerEnv: Map[String, String] = synchronized {
     assert(MINIKUBE_EXECUTABLE_DEST.exists(), EXPECTED_DOWNLOADED_MINIKUBE_MESSAGE)
-    executeMinikube("docker-env")
+    executeMinikube("docker-env", "--shell", "bash")
```

Comment:
Also added this because I switched to a different shell and it was giving me odd errors because our string replacement relies on bash style output